scripts/setup-grafana.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 <instance> <project>"
+    echo "Error: Both 'instance' and 'project' arguments are required."
+    exit 1
+fi
+
+INSTANCE=$1
+PROJECT=$2
+
```

Comment:
This will come handy later on.
```suggestion
SERVER_NAME="$(lxc info | awk '{if ($1 == "server_name:") print $2}')"
```